test/test_pebble.py

```diff
@@ -893,6 +894,134 @@ def build_mock_change_dict(change_id='70'):
     }
 
 
+class TestMultipartParser(unittest.TestCase):
+    class _Case:
```

Comment:
I was thinking a `collections.namedtuple` would be simpler for this, but I guess it's tricky to do a default (for `max_boundary`) with a namedtuple...

Reply:
Yeah - I don't feel strongly any particular way here, but the class does give better options for defaults and such.